contrib/ntp/ntpd/ntpd.c

```diff
@@ -1057,7 +1057,16 @@ ntpdmain(
 	/* Setup stack size in preparation for locking pages in memory. */
 # if defined(HAVE_MLOCKALL)
 #  ifdef HAVE_SETRLIMIT
+#   ifndef __CHERI_PURE_CAPABILITY__
+	/*
+	 * XXX: CheriABI currently puts ps_strings etc at the top of the stack
+	 * then rounds down to the next representable length, leaving huge
+	 * amoounts of padding. This means any moderately-restrictive limit
+	 * will cause us to segfault as soon as we touch any part of our stack.
+	 * Disable it entirely for now until we have a solution.
+	 */
```

Comment:
I guess we should link the changed lines in https://github.com/CTSRD-CHERI/cheribsd/issues/560 so we don't forget to remove this once it's fixed. Although I guess it shows up in our diff to upstream so shouldn't matter too much.

Reply:
The two are now forever linked by that comment :)